From linux-usb-devel-bounces@lists.sourceforge.net Sun Mar 11 13:04:45 2007
Date: Sun, 11 Mar 2007 20:03:03 +0000
From: Jon Dowland <jon@alcopop.org>
To: linux-usb-devel@lists.sourceforge.net
Message-ID: <20070311200303.GB4299@konishi.dowland>
Cc: Peter Korsgaard <jacmet@sunsite.dk>, linux-kernel@vger.kernel.org
Subject: USB: two more device ids for dm9601 usbnet driver

From: Jon Dowland <jon@alcopop.org>

This patch for the linux-usb-devel tree adds two more
product ids to the dm9601 driver. These ids were found on
rebadged dm9601 devices in the wild.

Signed-off-by: Jon Dowland <jon@alcopop.org>
Acked-by: Peter Korsgaard <jacmet@sunsite.dk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/net/dm9601.c |    8 ++++++++
 1 file changed, 8 insertions(+)

--- a/drivers/usb/net/dm9601.c
+++ b/drivers/usb/net/dm9601.c
@@ -578,6 +578,14 @@ static const struct usb_device_id produc
 	 USB_DEVICE(0x0a46, 0x9601),	/* Davicom USB-100 */
 	 .driver_info = (unsigned long)&dm9601_info,
 	 },
+	{
+	 USB_DEVICE(0x0a46, 0x6688),	/* ZT6688 USB NIC */
+	 .driver_info = (unsigned long)&dm9601_info,
+	 },
+	{
+	 USB_DEVICE(0x0a46, 0x0268),	/* ShanTou ST268 USB NIC */
+	 .driver_info = (unsigned long)&dm9601_info,
+	 },
 	{},			// END
 };
 
